chess_gui.py

```diff
@@ -119,12 +137,45 @@ def main():
     valid_moves = []
     game_over = False
 
+    knight_steps = 0
+    _is_check = 0
+    valid = False
+
     ai = ai_engine.chess_ai()
     game_state = chess_engine.game_state()
+    if chess_engine.game_state.whose_turn:
+        logger.info('white started')
+    else:
+        logger.info('black started')
+
     if human_player is 'b':
+        board = "\n---board---\n"
+        for i in range(8):
+            for j in range(8):
+                if game_state.is_valid_piece(i, j):
+                    board += game_state.get_piece(i, j).get_name()
+                    board += " "
+                else:
+                    board += Player.EMPTY
+                    board += " "
+            board += '\n'
+        logger.info(board)
         ai_move = ai.minimax_black(game_state, 3, -100000, 100000, True, Player.PLAYER_1)
+        if game_state.get_piece(ai_move[0][0], ai_move[0][1]).get_name() == 'n':
+            knight_steps += 1
         game_state.move_piece(ai_move[0], ai_move[1], True)
 
+    board = "\n---board---\n"
+    for i in range(8):
+        for j in range(8):
+            if game_state.is_valid_piece(i, j):
+                board += game_state.get_piece(i, j).get_name()
+                board += " "
+            else:
+                board += Player.EMPTY
+                board += " "
```

Comment:
Try to avoid i and j if better names are avalible

Reply:
I fixed it